pyflink-decodable/README.md

```diff
@@ -0,0 +1,64 @@
+# Getting Started With PyFlink On Decodable
+
+This example project shows how to run PyFlink jobs on Decodable.
+
+By deploying your PyFlink jobs as [https://docs.decodable.co/pipelines/create-pipelines-using-your-own-apache-flink-jobs.html](Custom Pipelines) onto Decodable, you can solely focus on implementing your job,
+while leaving all the aspects of running the job, like provisioning Flink clusters and the underlying hardware,
+keeping them secure and up-to-date, scaling them, monitoring and observing them, to the fully-managed Decodable platform.
+
+## Prerequisites
+
+You'll need the following things in place to run this example:
+
+* A free Decodable account ([https://app.decodable.co/-/accounts/create](sign up))
+* The [Decodable CLI](https://docs.decodable.co/cli.html)
```

Comment:
Happened to see this: in MD links it goes `[text](url)` (so as you have it in the second line here, but not the first).  There's at least one other place in this file with this mistake.
```suggestion
* A free Decodable account ([sign up](https://app.decodable.co/-/accounts/create))
* The [Decodable CLI](https://docs.decodable.co/cli.html)
```

Reply:
Argh, it happens _all the time_! Thx for fixing!